### PR TITLE
Add ClipboardEvent type

### PR DESCRIPTION
**Pull request by themoritz**
The clipboard and drag events now both share the `DOM.HTML.Event.DragEvent.DataTransfer` module, which might be weird. So maybe that module should be renamed? On the other hand this would be breaking...

### CODE REVIEW

File: src/DOM/HTML/Event/DragEvent/DataTransfer.purs [hunk 1]
```diff
@@ -25,3 +29,23 @@ foreign import filesNullable :: DataTransfer -> Nullable FileList
 -- | Returns an array of data formats used in the drag operation.
 -- | If the drag operation included no data, then the array is empty.
 foreign import types :: DataTransfer -> Array String
+
+-- | Retrieves the data for a given format, or an empty string if data for that
+-- | format does not exist or the data transfer object contains no data.
+foreign import getData
+  :: forall eff
+   . String
```
Comment (fehrenbach) [inline src/DOM/HTML/Event/DragEvent/DataTransfer.purs]: How about `MediaType` from purescript-media-types? It's already used in DOM.File.{Blob,File}.

**Comment by garyb** [comment]
> So maybe that module should be renamed? On the other hand this would be breaking...

We could move the stuff, and then just re-export from `DOM.HTML.Event.DragEvent.DataTransfer` - that'd prevent it from being a breaking change :)

There'll be a mass library release for 0.11, so I'll remove the old module then, if we do it that way.

**Comment by themoritz** [comment]
@garyb good idea! Any preferences where the `DataTransfer` module should live from now? `DOM.HTML.Event.DataTransfer`, or `DOM.HTML.DataTransfer`?

**Comment by garyb** [comment]
I was thinking `DOM.HTML.Event.DataTransfer`, but I don't know if it's used anywhere else? `DOM.HTML.DataTransfer` is fine too, I don't really mind :smile:

**Comment by themoritz** [comment]
I think it's not used outside of any event related stuff, so let's keep it under `Event`. :)

**Comment by garyb** [comment]
Looks great, thanks!

**Pull request merged**